mcbackend/__init__.py

```diff
@@ -13,3 +13,17 @@
     pass
 
 __version__ = "0.5.0"
+__all__ = [
+    "NumPyBackend",
+    "Backend", 
+    "Chain", 
+    "Run", 
+    "ChainMeta",
+    "Coordinate", 
+    "DataVariable", 
+    "ExtendedValue", 
+    "RunMeta", 
+    "Variable", 
+    "clickhouse", 
+    "ClickHouseBackend"
+    ]
```

Comment:
```suggestion
    "ClickHouseBackend",
]
```

Reply:
Good concern, but I've checked and we don't need to do that: 
- mypy doesn't complain about it
- the entries in `__all__` are strings and not the objects themselves, so that doesn't raise an error
- I've tested this at runtime, and get an error when I try to explicitly import `clickhouse`, but not otherwise.
<img width="1008" alt="Screenshot 2023-05-08 at 12 40 59" src="https://user-images.githubusercontent.com/2721423/236803801-05d864f7-2a43-421b-8eaa-c8a7391ffb21.png">